src/queue.c

```diff
@@ -23,6 +23,10 @@
 #include "protocol.h"
 #endif
 
+#if defined(__linux__)
```

Comment:
this kind of `#include` should be in `src/internal.h` with other dependencies that dispatch needs for most of its source.